.github/ISSUE_TEMPLATE/60-tflite-converter-issue.md

```diff
@@ -5,42 +5,50 @@ labels: 'TFLiteConverter'
 
 ---
 
+### 1. System information
 
-**System information**
 - OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
 - TensorFlow installed from (source or binary):
 - TensorFlow version (or github SHA if from source):
 
+### 2. Code (that demonstrates how to reproduce your issue)
+**Option A: Using reference colab notebooks**
 
-**Command used to run the converter or code if you’re using the Python API**
-If possible, please share a link to Colab/Jupyter/any notebook.
+The reference colab notebooks given below demonstrate: *(TensorFlow Model Colab)* Build model in TF Keras --> *(TensorFlow Lite Model Colab)* Convert to TF Lite (performing quantization techniques) and run TFLite Inference. You may use these colab notebooks as a reference point to generate your model behavior and attach links.
+
+1)  Reference [TensorFlow Model Colab](https://colab.research.google.com/gist/ymodak/e96a4270b953201d5362c61c1e8b78aa/tensorflow-datasets.ipynb?authuser=1)
+2)  Reference [TensorFlow Lite Model Colab](https://colab.research.google.com/gist/ymodak/0dfeb28255e189c5c48d9093f296e9a8/tensorflow-lite-debugger-colab.ipynb)
 
```

Comment:
Those are nice colab links. How about keeping them somewhere in TensorFlow Lite repository?